### DISCUSSION TITLE
How do I limit the sites my extension runs on?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [antfu/vitesse-webext#182] [Q&A] [answered]
How can I limit the sites my extension runs on? 

I've attempted the following In my `src/manifest.ts` file:

```ts
content_scripts: [
      {
        matches: [
          'https://twitter.com/*',
          'https://mobile.twitter.com/*',
          'https://x.com/*',
        ],
        js: [
          'dist/contentScripts/index.global.js',
        ],
      }
]
```

But, it doesn't work. My extension is still loading and applying its effect on other tabs/domains.

**Comment by tmkx** [comment, marked as answer]
https://developer.chrome.com/docs/extensions/develop/concepts/content-scripts#static-declarative

have you reloaded the extension after you changed the `matches` rules?

**Reply by CodyBontecou** [reply]
Hmm. It seems like something happened to my dev server. Restarting it did the trick.
